config/routes.rb

```diff
@@ -12,7 +12,7 @@
 
   mount RailsAdmin::Engine => '/admin', as: 'rails_admin'
 
-  resources :paperworks, :case_notes, :professional_questionnaires, :personal_questionnaires, only: [:index, :show, :new, :edit]
+  resources :assignments, :paperworks, :case_notes, :professional_questionnaires, :personal_questionnaires, only: [:index, :show, :new, :edit]
```

Comment:
Your also have resources assignments in line 45. Delete one of these.

Reply:
Correct me if I'm wrong but I believe the other resources :assignments is underneath the namespace :api, which makes both of them necessary.

Reply:
ah yes my bad good catch